frontend/src/layout/navigation/SideBar/SideBar.scss

```diff
@@ -58,6 +58,13 @@
     > div {
         margin: 0.5rem 0;
     }
+
+    .Pages {
+        > .LemonButtonWithSideAction,
+        > a button {
+            margin-bottom: 1px;
+        }
+    }
```

Comment:
I'd love it if we could do this _without_ using CSS. That way we can change the internals of the LemonButton without actually breaking this code. 

Best would be in the actual wrapper for the sidebar buttons adding the 1px gap. We might need a new utility class like `space-y-px` as I don't think pixel level separation is possible

Reply:
Made the change and opted for a unique CSS still (as we don't have many other cases of pixel spacing) but wrapping the buttons rather than modifying them directly